charts/timed/values.yaml

```diff
@@ -221,7 +221,7 @@ postgresql:
   # postgresqlPassword: s3cr3t
   image:
     # postgresql.image.tag -- PostgreSQL image version to use
-    tag: 12.2.0
+    tag: 12.3.0
```

Comment:
Can you add a quick comment as to why we are overriding the image tag and when we can get rid of the override?

Other than that, LGTM!

Reply:
Added a comment. The bitnami chart uses Postgresql 11, so we probably can't get rid off it anytime soon, or am I missing something?

Reply:
I can't seem to find the comment :/ I should tell us why we are using PostgreSQL 12 and maybe hint at the fact that we can remove the tag override once the chart has official support for 12.x